src/App.spec.tsx

```diff
@@ -0,0 +1,8 @@
+import {render} from 'react-dom';
```

Comment:
you think its better to keep specs in the same directory as the component instead of a separate directory?

Reply:
im ok with either

Reply:
I slightly prefer keeping them in the same dir, i guess.